src/test/java/org/folio/innreach/controller/d2ir/PatronInfoControllerTest.java

```diff
@@ -304,6 +307,99 @@ void return200HttpCode_and_patronInfoResponseWithPatronInfo_hasIgnoreCaseCorrect
     assertNotNull(response.getPatronInfo());
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {"jo hn do e","do e jo hn","jo hn do e pauL","do e Jo hn pauL","do e, Jo hn, pauL", "do e Jo hn, pauL"})
+  @Sql(scripts = {
+    "classpath:db/central-server/pre-populate-central-server.sql",
+    "classpath:db/patron-type-mapping/pre-populate-patron-type-mapping.sql",
+    "classpath:db/user-custom-field-mapping/pre-populate-user-custom-field-mapping.sql"
+  })
+  void return200HttpCode_and_patronInfoResponseWithPatronInfo_WithSpaceInFirstAndLastName(String patronName) {
+    // This test covers "first first last last", "last last first first", "first first last last middle" "last last, first first middle"
+    var user = createUserWithSpacesInFirstAndLastName("jo hn", "do e");
```

Comment:
Shouldn't this be something like `createUserWithTwoFirstAndTwoLastNames` just to be more accurate?

Reply:
Changed the name as per the suggestion